recipes/pffft/all/test_package/test_package.c

```diff
@@ -0,0 +1,9 @@
+#include <pffft.h>
+#include <stdio.h>
+
+int main() {
+
+    int s = pffft_simd_size();
+    printf(s);
```

Comment:
```suggestion
    printf("pffft_simd_size: %d\n", s);
```

Reply:
It's fine now. But this was embarrassing... I swear I can code!